foia_hub/templates/500.html

```diff
@@ -1,17 +1,34 @@
 {% extends "base.html" %}
 
+
 {% block body %}
-<div class="container">
-    <section class="content">
-        <h1> 500. Error. </h1>
+<div id="main">
+  <div class="search search__error">
+    <div class="container">
+        <section class="content">
+            <h1><em>500. Error.</em></h1>
+
+            <h2>We're sorry, something went wrong.</h2>
+
+            <p>If you're looking for a FOIA contact information, try searching:</p>
 
-        <p> 
-            Looks like something went wrong. 
-        </p>
+            <form class="agency" method="get" action="/agencies/">
```

Comment:
Not familiar with the circumstances in which this error is shown, so open to whatever copy makes the most sense.

# 500. Error.

## We're sorry, something went wrong.

Try reloading, or check again later.